gravitybridge/assetlist.json

```diff
@@ -20,11 +20,48 @@
         "name": "Graviton",
         "display": "graviton",
         "symbol": "GRAV",
-	"coingecko_id": "graviton",
+  	    "coingecko_id": "graviton",
         "logo_URIs": {
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/gravitybridge/images/grav.svg",
-	  "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/gravitybridge/images/grav.png"
+	        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/gravitybridge/images/grav.png"
         }
+      },
+      {
+        "description":
+          "pSTAKE is a liquid staking protocol unlocking the liquidity of staked assets.",
+        "denom_units": [
+          {
+            "denom": "gravity0xfB5c6815cA3AC72Ce9F5006869AE67f18bF77006",
+            "exponent": 0
+          }
```

Comment:
yep, needs another denom units defined here:
```
{
  "denom": "pstake",
  "exponent": 6
}
```
I'm guessing it's 6

Reply:
oo oo it's 18!!